configurations/morcore2esuites/Configuration_TaakApiInt.xml

```diff
@@ -0,0 +1,161 @@
+<Module xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="../FrankConfig.xsd">
+    <Adapter name="NewResponse_Notification">
+        <Receiver name="Response_Notification">
+            <JavaListener name="Response_Notification"/>
+        </Receiver>
+
+        <Pipeline>
+
+            <Exits>
+                <Exit name="Exit" state="SUCCESS"/>
+                <Exit name="failure" state="REJECTED"/>
+            </Exits>
+
+        <SenderPipe name="Send2MorCore">
+            <HttpSender name="GetNotification" methodType="GET" url="/melding/{uuid}"/>
+            <Forward name="success" path="Json2Xml"/>
+        </SenderPipe>
+
+        <JsonPipe name="Json2Xml" direction="JSON2XML" prettyPrint="true">
+            <Forward name="success" path="JuisteFormat"/>
+            <Forward name="Exception" path="failure"/>
+        </JsonPipe>
+
+        <XsltPipe name="JuisteFormat" styleSheetName="xsl/TaakApiInt/GetNotification.xsl" outputType="XML">
+            <Forward name="success" path="XmlCheck"/>
+            <Forward name="Exception" path="failure"/>
+        </XsltPipe>
+
+        <XmlValidatorPipe name="XmlCheck" schema="xsd/TaakApiInt/ValidateGetMelding.xsd" root="root">
+            <Forward name="success" path="Xml2Json"/>
+            <Forward name="Exception" path="failure"/>
+        </XmlValidatorPipe>
+
+        <JsonPipe name="Xml2Json" direction="XML2JSON">
+            <Forward name="success" path="Exit"/>
+            <Forward name="Exception" path="failure"/>
+        </JsonPipe>
+
+        <!-- Mogelijke config update mogelijk ZIE HIERONDER!
```

Comment:
Consider changing this comment to explain why the code below is not currently enabled and when it should become enabled. We don't want to end up forgetting. ;)